### ISSUE TITLE
Move all works from hotfolder to importing on import start

### ISSUE THREAD

**Issue by ponchofiesta** [tuub/kitodo-mediaserver#256]
Actually we can not export from Kitodo.Production and import into Mediaserver at the same time because both are using the same folder. When the importer starts, it could move all works from hotfolder to importing and process them. So the hotfolder could accept new exports from Production.

**Comment by pbroman** [comment]
I don't think this would solve the problem. If you move all works and there is an export in progress, you will only move half the work and break the export.
The present solution is, the at the importer only considers works with a mets file, moves them one at the time to the importing area and performs the import. If a folder has no mets file, it is left to the next import, and I think, this is the best we can do.
